san/sanbase_graphql_helper.py

```diff
@@ -271,11 +271,17 @@ def _format_from_date(datetime_obj_or_str):
 
 
 def _format_to_date(datetime_obj_or_str):
-    if isinstance(datetime_obj_or_str, datetime.datetime):
-        datetime_obj_or_str = datetime_obj_or_str.isoformat()
+    date_format = '%Y-%m-%d'
+    if not isinstance(datetime_obj_or_str, datetime.datetime):            
+        try:
+            datetime.datetime.strptime(datetime_obj_or_str, date_format)
```

Comment:
date_format is used only here - you can just use
```python
datetime.datetime.strptime(datetime_obj_or_str, '%Y-%m-%d')
```

Reply:
Yeah, you're right